Database/Corrections/sodQuestFixes.lua

```diff
@@ -2963,6 +2963,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.objectivesText] = {"Kneel at the Heretic Idol and receive the rune."},
             [questKeys.requiredSpell] = -402849,
         },
+        [90190] = {
+            [questKeys.name] = "Shadow Word: Death",
+            [questKeys.startedBy] = {nil,{411674}},
```

Comment:
The spawn needs to be cleaned up.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5570/commits/c89fea46e55e22d689c5752c16c44498eae4f59d